community/kernel/src/main/java/org/neo4j/kernel/api/proc/Procedure.java

```diff
@@ -36,7 +37,7 @@
 public interface Procedure
```

Comment:
Agreed - they also need to be moved to a public API package. We'll do that in a follow-up as we iterate on this.